q_setup.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Run this script to set up the qsim environment for the first time.
+# You can read the following steps to see what each is doing.
+#
+# Author: Pranith Kumar
+# Date: 01/05/2016
+# Usage: ./setup.sh {arm64}
+
```

Comment:
This file is not realy necessary. I have create this for the QT-Creator. 

I use the QT-Creator as a IDE. If you like I can share with you my IDE-Files